app/models/ability.rb

```diff
@@ -93,7 +93,7 @@ def calculate_permissions(user)
 
   def to_list
     # Skipping rules that only contain a block.
-    rules.select { |r| !r.only_block? }.map do |rule|
+    rules.reject(&:only_block?).map do |rule|
       rule.only_block?
```

Comment:
could you explain what this line (97) is for? thanks

Reply:
That is an excellent question.  That line appears to do...absolutely nothing!  Haha

I looked back at the commit that originally added it (from a couple of years ago) and it looks like it was accidentally left in.  I'll remove it and verify that the tests still pass.

Thank you for catching that!

Reply:
Tests passed!